src/app/components/Transaction/index.tsx

```diff
@@ -428,7 +429,7 @@ export function Transaction(props: TransactionProps) {
         </Text>
       </CardHeader>
       <StyledCardBody margin={{ bottom: 'small' }}>
-        <Box width="75%">
+        <Box>
```

Comment:
just a personal preference, prev version looks a bit better on desktop (when tx details adopt to screen width)
we could add  `style={{ flex: 1 }}` here and make attr in L~447 `width="max-content"` conditional. 
Did not test it fully and personally don't care that much about it.

Reply:
I didn't find a good way to prevent wobbly layout yet
![image](https://github.com/user-attachments/assets/4e126f4b-a525-4380-bcf6-7e7f310d4833)

Reply:
Ignoring for now